util/collect_coverage.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -euo pipefail
+set -x
+
+echo here!
+
+# export LLVM_PROFILE_FILE="${COVERAGE_DIR}/%h-%p-%m.profraw"
+output_file=$COVERAGE_DIR/_cc_coverage.dat
+
+/Users/ksmiley/dev/rules_rust/examples/bazel-examples/external/rust_darwin_aarch64/lib/rustlib/aarch64-apple-darwin/bin/llvm-profdata \
```

Comment:
So I think in general all we have to do is make this script use the right tools, and output the expected file path